core/org.wso2.carbon.user.core/src/main/java/org/wso2/carbon/user/core/UniqueIDUserStoreManager.java

```diff
@@ -158,6 +158,33 @@ AuthenticationResult authenticateWithID(List<LoginIdentifier> loginIdentifiers,
      */
     List<User> getUserListOfRoleWithID(String roleName, String filter, int maxItemLimit) throws UserStoreException;
 
+    /**
+     * Get user list of group.
+     *
+     * @param groupName Name of the group.
+     * @return An array of users that belongs to the given group.
+     * @throws UserStoreException Thrown by the underlying UserStoreManager.
+     */
+    default List<User> getUserListOfGroupWithID(String groupName) throws UserStoreException {
+
+        throw new NotImplementedException(
+                "getUserListOfGroupWithID operation is not implemented in: " + this.getClass());
+    }
+
+    /**
+     * Get user list of group.
+     *
+     * @param groupName Name of the group.
+     * @return An array of users that belongs to the given group.
+     * @throws UserStoreException Thrown by the underlying UserStoreManager.
```

Comment:
filter and maxItemLimit params are not described in the comment

Reply:
Addressed with https://github.com/wso2/carbon-kernel/pull/4003